### PR TITLE
API serves requests with categories

### PR DESCRIPTION
**Pull request by kermit-klein**
[PT LINK](https://www.pivotaltracker.com/story/show/173432919)

- Add index action to include category parameter
- Update factories,models

### CODE REVIEW

File: app/serializers/my_request/quest/index_serializer.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class MyRequest::Quest::IndexSerializer < ActiveModel::Serializer
+  attributes :id, :title, :reward
```
Comment (jysmys) [inline app/serializers/my_request/quest/index_serializer.rb:4]: I thought that the quest should have a status as well ? true : false

Comment (jysmys) [reply app/serializers/my_request/quest/index_serializer.rb:4]: false &&
{message: "I understand", status: 200}

**Review by jysmys — APPROVED** [review]
Good work Ali

**Comment by PaulineBA** [comment]
Ali, you ARE the king of the perfect commit log!!!

**Review by PaulineBA — APPROVED** [review]

**Review by designerofthing — APPROVED** [review]

**Pull request merged**